src/mui/input/DateInput.js

```diff
@@ -7,8 +7,7 @@ export const datify = input => {
         return null;
     }
 
-    let date = input instanceof Date ? input : new Date(input);
-	date = new Date(date.getUTCFullYear(), date.getUTCMonth(), date.getUTCDate());
+    let date = input instanceof Date ? input : new Date(input.replace('-','/','g'));
```

Comment:
nice, but you should do that in the other place, too. And switch back to `const`